docs/book/src/usage/ci.md

```diff
@@ -0,0 +1,125 @@
+# CI
+
+Marker's explicit goal is to make it easy to use on CI. This document describes the provided CI tooling and demonstrates example templates.
+
+<!-- toc -->
+
+## GitHub Action
+
+Marker provides a GitHub Action that downloads the pre-compiled binaries and runs `cargo marker`.
+
+<!-- region replace-version stable -->
+
+```yml
+- uses: rust-marker/marker@v0.2
+```
+
+### Git tags
+
+The git tag used for the GitHub Action selects the version of Marker that will be installed. There are several flavours of git tags:
+
+- Sliding `v0.2` *(recommended)*. Use this to get automatic patch updates.
+- Fixed `v0.2.1`. Use this if you want to pin a specific version if there is a regression in a new patch version. If there is one, please create a [new issue], patch versions must never break anything!
+
+<!-- endregion replace-version stable -->
+
+> ⚠️ Different `v0.{minor}` versions contain breaking changes. There is `v0` git tag, but because there are breaking changes on each minor release it doesn't make sense to use it. However once Marker reaches `1.0` minor versions will be compatible and `v1` tag will be stable.
```

Comment:
Formulation suggestion:

```suggestion
> ⚠️ The minor versions before Marker `v1` contain breaking changes. While there is a sliding `v0` tag, it's highly recommended to include the minor version as well. This prevents uncontrolled CI breakage with every release
```

Side note: I plan to write a chapter on stability, once v0.3.0 has been released.